src/components/mainPage/JobSelectionSection.jsx

```diff
@@ -38,13 +38,13 @@ const JobSelectionSection = forwardRef((props, ref) => {
           <JobButton
             icon={LiveIcon}
             jobType="LIVE JOB"
-            description="See results in real time. Up to 10 images and videos"
+            description="Submit up to 10 images and videos and watch the results appear in real time"
```

Comment:
Not a fan of the wording of this

Reply:
feel free to suggest any

Reply:
How about: "Submit up to 10 images and videos and see the results in real time"?

Reply:
`see` doesn't sound good, how about `Submit up to 10 images and videos and watch the results in real time` 

```
To see means to be aware of what is around you by using your eyes.

For example: "I can see the smoke from here."

To watch means to deliberately look at something for a period of time, especially something that is changing or moving.

For example: "I watched the cricket."
```

Reply:
I don't think `watch` sounds good

Reply:
What about `"...view the results in real time"?`